backend/src/helpers/matches.js

```diff
@@ -19,7 +19,12 @@ exports.patchMatchStatus = async function (MatchID, MatchStatus) {
   try {
     let query = await db.Match.query()
       .findById(MatchID)
-      .patch({ MatchStatus: MatchStatus });
+      .patch(
+        {
+           MatchStatus: MatchStatus,
+           ConfirmedDate: new Date()
```

Comment:
Do we update the `ConfirmedDate` no matter what the updated `MatchStatus` is? Or only in certain cases (not been following this feature so not sure how it works).

Reply:
As discussed, currently the `MatchStatus` can only be toggled from "Pending" to "Active". Marking as resolved.